example/gluon/audio/urban_sounds/README.md

```diff
@@ -1,100 +0,0 @@
-# Urban Sounds Classification in MXNet Gluon
```

Comment:
Why is this removed?

Reply:
Sorry, I don't know anything about this file, I just accept this change which made by communities before committing my branch.